tex/latex/fks/fksmeta.sty

```diff
@@ -80,9 +80,35 @@
 \met@probheader & %
 \Roman{solvedbatch} & \textnormal{\%} & $\Sigma$ \\}
 
+% For switching between FYKOS and Vyfuk functionality across texmf files.
```

Comment:
Používají se tyhle ify někde?

Chtěl bych vystačit jen s optionami, potažmo meta-makry.